controllers/topic.controller.js

```diff
@@ -15,3 +16,7 @@ exports.postTopic = (req, res, next) => {
         next(err);
     })
 }
+
+exports.getAllEndpoints = (req, res) => {
+    res.status(200).send(endpointsFile)
+}
```

Comment:
Good work moving this logic into the controller, `getAllEndpoints` is a good function name and sending the response as is using the required json is good.